Documentation/ContentObjects/Fluidtemplate/DataProcessing.rst

```diff
@@ -205,6 +205,31 @@ In the Fluid template then iterate over the files:
       </f:for>
    </ul>
 
+
+Example: use stdWrap property on references
+-------------------------------------------
+
+.. versionadded:: 10.3
+
+   :ref:`stdWrap` properties added to the :php:`FilesProcessor`.
+
+The following example implements a slide-functionality on rootline
+for file resources::
+
+   page.10.dataProcessing {
+      10 = TYPO3\CMS\Frontend\DataProcessing\FilesProcessor
+      10 {
+         references.data = levelmedia: -1, slide
+         as = myfiles
+      }
+   }
+
+The :php:`FilesProcessor` can slide up and down the rootline to collect
+images for FLUID templates. One usual feature is to use images
+attached to pages and use them up and down the page tree for
```

Comment:
Is "up and down the page tree" misleading? I would assume images are only looked for by going up the pagetree?

Reply:
You are looking for images up the pagetree, however the images get slided down the rootline. Graa!

Reply:
Yes, I understand. It is a bit difficult to explain.